erigon-lib/chain/chain_config.go

```diff
@@ -198,6 +198,14 @@ func (c *Config) IsShanghai(time uint64) bool {
 	return isForked(c.ShanghaiTime, time)
 }
 
+// IsAgra returns whether num is either equal to the Agra fork block or greater.
+func (c *Config) IsAgra(num uint64) bool {
+	if c == nil || c.Bor == nil {
+		return false
+	}
+	return isForked(c.Bor.AgraBlock, num)
```

Comment:
Let's use `c.Bor.IsAgra(num)`

So that if we change tomorrow something in `IsAgra` of bor, It gets reflected here automatically.